src/SFML/Window/CMakeLists.txt

```diff
@@ -203,6 +204,7 @@ elseif(SFML_OS_MACOSX)
     )
     source_group("mac" FILES ${PLATFORM_SRC})
 elseif(SFML_OS_IOS)
+    enable_language(OBJCXX)
```

Comment:
What's the advantage of splitting things up, compared to https://github.com/SFML/SFML/commit/90af02f318655dd51c3bf640d7c37582f751e2e0 ?

Reply:
Less maintenance how?

Reply:
Wrt the conditionals. It's easier to keep things in sync when the language is enabled at the usage site.